Scripts/assignment/emme_assignment.py

```diff
@@ -376,6 +376,38 @@ def aggregate_results(self,
                         miles["dist"][mode] += departures * segment.link.length
                         miles["time"][mode] += departures * segment[time_attr]
         resultdata.print_data(miles, "transit_kms.txt")
+        network = self.day_scenario.get_network()
```

Comment:
I think this should be in an own function instead of extending the already too long aggregate_results() method